### PR TITLE
Added transmission implementation for broadcast type groups

### PR DESCRIPTION
**Pull request by ethouris**

### CODE REVIEW

**Comment by ethouris** [comment]
NOTE: Tests do not fail, it was a test env problem.

File: srtcore/api.cpp [hunk 1]
```diff
@@ -746,6 +746,8 @@ int CUDTUnited::newConnection(const SRTSOCKET listen, const sockaddr_any& peer,
 
       CTimer::triggerEvent();
 
+      CTimer::triggerEvent();
+
       // XXX the exact value of 'error' is ignored
```
Comment (maxsharabayko) [inline srtcore/api.cpp]: Please check the PR for similar errors.

File: srtcore/core.cpp [hunk 2]
```diff
@@ -3362,6 +3364,41 @@ bool CUDT::interpretGroup(const int32_t groupdata[], int hsreq_type_cmd SRT_ATR_
...
+    // Synchronize the TSBPD PEER start time with the existing connection,
+    // if there exists the connection with given peer.
+    if (master_peerid != -1)
+    {
+        // Here "I am a peer", so this is the socket ID of local socket of a parallel connection.
+        // Check if it exists, if not, reject the connection.
+        CUDTSocket* master = s_UDTUnited.locateSocket(master_peerid, s_UDTUnited.ERH_RETURN);
+        if (!master)
+        {
+            LOGC(mglog.Error, log << "HS/GROUP: master parallel connection socket not found: $" << master_peerid);
```
Comment (maxsharabayko) [inline srtcore/core.cpp]: This was already removed from previous PR...

File: srtcore/core.cpp [hunk 3]
```diff
@@ -11419,6 +11419,593 @@ int CUDTGroup::send(const char* buf, int len, SRT_MSGCTRL& w_mc)
...
+            if (!d->ps || d->ps->getStatus() == SRTS_BROKEN)
+            {
+                HLOGC(dlog.Debug, log << "... permanently. Will delete it from group $" << id());
+                wipeme.push_back(d);
+            }
+            else
+            {
+                HLOGC(dlog.Debug, log << "... socket still " << SockStatusStr(d->ps ? d->ps->getStatus() : SRTS_NONEXIST));
+            }
+            */
```
[the remark this answers is not in the document]
Comment (ethouris) [reply srtcore/core.cpp]: Again????

[the remark this answers is not in the document]
Comment (maxsharabayko) [reply srtcore/core.cpp]: What again?

**Pull request merged**